### PR TITLE
Enhancement/4106 loading state on secondary ga4 dropdown

### PR DESCRIPTION
**Pull request by eugene-manuilov**
## Summary

Addresses issue #4106

## Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [ ] I have added a QA Brief on the issue linked above.
- [ ] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

**Review by tofumatt — CHANGES_REQUESTED** [review]
Looks good, but can you add a QA Brief for this issue?

**Comment by eugene-manuilov** [comment]
@tofumatt QAB is added.

**Review by tofumatt — CHANGES_REQUESTED** [review]
It looks like there's a merge conflict here that needs resolving.

Regarding the QA brief: it might be missing some steps? I wasn't able to see a secondary dropdown as mentioned in the issue when doing Analytics setup with or without GA4setup enabled. I see this in the settings, when switching accounts:



Is something wrong with my build or am I missing a step to get to the point where this scenario comes up?

**Comment by eugene-manuilov** [comment]
@tofumatt you need to disconnect the Analytics module and setup it again with UA+GA4 properties selected. Then you will see GA4 dropdown in the settings form and will be able to observe the progress bar for the GA4 dropdown.

Merge conflict is resolved.

File: assets/js/components/Header.stories.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ import Header from './Header';
 import DateRangeSelector from './DateRangeSelector';
 import HelpMenu from './help/HelpMenu';
 import HelpMenuLink from './help/HelpMenuLink';
-import UserInputSuccessNotification from './notifications/UserInputSuccessNotification';
+import UserInputSuccessBannerNotification from './notifications/UserInputSuccessBannerNotification';
```
Comment (eugene-manuilov) [inline assets/js/components/Header.stories.js:32]: I have had to make these changes after merging the develop branch into mine.

**Review by tofumatt — APPROVED** [review]
Thanks for the extra context on the QA process. Got it working and looks good, so I'll get this merged as soon as all of the E2E tests run. 👍🏻

**Pull request merged**